v2/cmd/wails/internal/commands/initialise/templates/generate/assets/lit-ts/frontend/src/my-element.ts

```diff
@@ -68,32 +68,31 @@ export class MyElement extends LitElement {
   }
 
     `
-}
 
-@property()
-resultText = "Please enter your name below 👇"
+    @property()
+    resultText = "Please enter your name below 👇"
 
-greet()
-{
-    let thisName = this.shadowRoot.getElementById('name').value
-    Greet(thisName).then(result => {
-        this.resultText = result
-    });
-}
+    greet() {
+        let thisName = (this.shadowRoot.getElementById('name') as HTMLInputElement)?.value;
+        if (thisName) {
+            Greet(thisName || 'UAAA').then(result => {
```

Comment:
'UAAA'?

Reply:
My bad, it was a bad null check ! Value and innerText returned null until I applie the HTMLInputElement type assertion. Fixed with latest commit